include/boost/math/special_functions/legendre.hpp

```diff
@@ -184,7 +184,7 @@ std::vector<T> legendre_p_zeros_imp(int n, const Policy& pol)
     }
     T half_n = ceil(n*half<T>());
 
-    while (k < (int)zeros.size())
+    while (k < zeros.size())
```

Comment:
I believe this change will result in a signed/unsigned comparison warning on msvc?